.env.test

```diff
@@ -0,0 +1,4 @@
+# define your env variables for the test env here
+KERNEL_CLASS='App\Kernel'
+APP_SECRET='s$cretf0rt3st'
+SYMFONY_DEPRECATIONS_HELPER=999999
```

Comment:
Actually we do not need this file, since we are not using phpunit symfony bridge

Reply:
Yea, it's generated flex spam. I didn't notice it, just remote it